releasenotes/notes/0.19/added-multiformat-support-b5d3c7c7c1536951.yaml

```
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Using :meth:`.QuantumCircuit.draw` with the ``latex`` drawer will now generate a file
```

We should also mention the standalone drawer function:

```suggestion
    Using :meth:`.QuantumCircuit.draw` or :func:`~qiskit.visualization.circuit_drawer`
    with the ``output`` argument set to ``"latex"`` the drawer will now generate a file
```